numpy/lib/stride_tricks.py

```diff
@@ -111,6 +112,211 @@ def as_strided(x, shape=None, strides=None, subok=False, writeable=True):
     return view
 
 
+def _sliding_window_view_dispatcher(x, window_shape, axis=None, *,
+                                    subok=None, writeable=None):
+    return (x,)
+
+
+@array_function_dispatch(_sliding_window_view_dispatcher, module='numpy')
+def sliding_window_view(x, window_shape, axis=None, *,
+                        subok=False, writeable=False):
+    """
+    Create a sliding window view into the array with the given window shape.
+
+    Creates a sliding window view (also known as rolling or moving window) of
+    the N dimensional array with the given window shape. The window slides
```

Comment:
It seems a bit odd to have this sentence repeated, the second time in slightly more detail. Not a blocker against merging, but maybe someone on the doc team can offer a better phrasing.